ipywidgets_bokeh/src/manager.ts

```diff
@@ -208,4 +211,33 @@ export class WidgetManager extends HTMLManager {
     }
     return super.new_model(options, serialized_state)
   }
+
+  protected override loadClass(
```

Comment:
This came from an aborted attempt to extend `ManagerBase` rather than `HTMLManager` which may be required due to changes in ipywidgets 8.